docs/okeps/okep-5094-layer2-transit-router.md

```diff
@@ -0,0 +1,757 @@
+       102 ip6.src == 2010:100:200::/60 && ip6.dst == 2010:100:200::/60           allow
+       102 ip6.src == 2010:100:200::/60 && ip6.dst == fd98::/64           allow
+       102                                     pkt.mark == 42           allow
+```
+
+And these the gateway router routes (there is no policies)
+```
+IPv4 Routes
+Route Table <main>:
+           169.254.0.0/17               169.254.0.4 dst-ip rtoe-GR_test12_namespace.scoped_ovn-control-plane
+           203.203.0.0/16               100.88.0.13 dst-ip rtotr-GR_test12_namespace.scoped_ovn-control-plane <-- inbound egress traffic goes back to the ovn_cluster_router peer ip.
+                0.0.0.0/0                172.18.0.1 dst-ip rtoe-GR_test12_namespace.scoped_ovn-control-plane
+```
+
+For the routes related to EIP they are moved from gateway router to ovn_cluster_router and the redirect points to the transit node peer ip.
+
+#### Router to router direct connection without a switch
+
+To connect the ovn_cluster_router to the local gateway router instead of using a join switch like the layer3 topology those ports can be connected each other directly
+using the `peer` field.
+
+This is how it looks the `trtor` port at ovn_cluster_router:
+```
+_uuid               : f697bce1-dac7-442d-9355-e298e1735c7b
+dhcp_relay          : []
+enabled             : []
+external_ids        : {"k8s.ovn.org/network"=test12_namespace-scoped, "k8s.ovn.org/topology"=layer2}
+gateway_chassis     : []
+ha_chassis_group    : []
+ipv6_prefix         : []
+ipv6_ra_configs     : {}
+mac                 : "0a:58:64:58:00:0d"
+name                : trtor-GR_test12_namespace.scoped_ovn-control-plane
+networks            : ["100.88.0.13/30", "fd97::8/127"]
+options             : {requested-tnl-key="4"}
+peer                : rtotr-GR_test12_namespace.scoped_ovn-control-plane <------------- peer field
+status              : {}
+```
+
+And this is the `rtotr` port at GR:
+```
+_uuid               : f9ab92f3-478c-41dd-b845-0d8ddf4a34e5
+dhcp_relay          : []
+enabled             : []
+external_ids        : {"k8s.ovn.org/network"=test12_namespace-scoped, "k8s.ovn.org/topology"=layer2}
+gateway_chassis     : []
+ha_chassis_group    : []
+ipv6_prefix         : []
+ipv6_ra_configs     : {}
+mac                 : "0a:58:64:41:00:04"
+name                : rtotr-GR_test12_namespace.scoped_ovn-control-plane
+networks            : ["100.65.0.4/16", "100.88.0.14/30", "fd97::9/127", "fd99::4/64"]
+options             : {gateway_mtu="1400"}
+peer                : trtor-GR_test12_namespace.scoped_ovn-control-plane <-------------- peer field
+status              : {}
+
+```
+
+#### Transit router specifics
+
+For transit router to work the LRPs referencing the same gateway router peer should have an unique tunnel key and if they are remote also a
+requested chassis pointing to the gateway router node.
+
+Also the LRP connected to the switch need a gateway router configured for NAT to work.
```

Comment:
"Also, the transit router port connected to the switch needs to be configured as gateway router port for the conditional SNAT to work."

Reply:
> Maybe also dump the NB configuration for the LRP here?

The tunneling diagram includes the gateway chasssis at the LRP.